lib/membrane_precompiled_dependency_provider.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Membrane.PrecompiledDependencyProvider do
+  @moduledoc """
+  Module providing URLs for precompiled dependencies used by Membrane plugins.
+
+  Dependencies that are fully located in the repositories of `membraneframework-precompiled` will
+  be referred to as Generic. Otherwise they will be referred to as Non-generic.
+  """
+  @membrane_precompiled_org_url "https://github.com/membraneframework-precompiled"
+
+  @non_generic_precompiled_deps [
+    :ffmpeg
+  ]
+
+  @type precompiled_dependency() ::
+          :ffmpeg | :portaudio | :"fdk-aac" | :srtp | :opus | :sdl2 | :portaudio | :mad
+
+  @doc """
+  Get URL of a precompiled build of given dependency for appropriate target.
+  """
+  @spec get_precompiled_dependency_url(dependency :: precompiled_dependency()) ::
+          String.t() | nil
+  def get_precompiled_dependency_url(dependency) do
```

Comment:
```suggestion
  def get_dependency_url(dependency) do
```

IMO it's clear from the module name that the dependency is precompiled ;)